### ISSUE TITLE
Latest release totally broken....

### ISSUE THREAD

**Issue by Stono** [roboll/helmfile#451]
```
❯ helmfile sync
read .: is a directory
```

```
monkeynetes/helmfile git/master
❯ cat helmfile.yaml
---
releases:
- name: monkeynetes
  namespace: monkeynetes
  chart: aws/platform-helm-at-service
```

**Comment by mumoshu** [comment]
Thanks for reporting! Would you mind sharing a full output with
--log-level=debug?
2019年1月24日(木) 18:50 Karl Stoney <notifications@github.com>:

**Comment by Stono** [comment]
```
❯ helmfile --log-level=debug sync
Processing helmfile.yaml
changing working directory to "/Users/karl.stoney/git/autotrader/monkeynetes/helmfile"
changing working directory back to "/Users/karl.stoney/git/autotrader/monkeynetes/helmfile"
read .: is a directory
```

**Comment by pdemagny** [comment]
I'm using v0.43.1 and for me it only works when using explicit `-f helmfile.yaml`, which is supposed to be the default value.

**Comment by mumoshu** [comment]
@Stono @pdemagny Just released v0.43.2 with the possible fix.

Could you try once again, and report back if it doesn't work yet? Thanks again for your support!

**Comment by pdemagny** [comment]
@mumoshu I confirm that it now works ! Thanks for this quick fix :+1:
